File: user_input/Rust/message.rs
/* Compile using rustc message.rs */
// Takes string input from user and writes to console.

use std::*;

fn main(){
    let mut input = String::new();
    println!("Write some text:");
    io::stdin().read_line(&mut input)
        .ok()
        .expect("error: unable to read user input");
    println!("Here's your text back:\n{}", input);
}

File: user_input/Fortran/message.f90
! Takes string input from user and writes it back to console
! Compile with gfortran message.f90 and run with ./a.out

program Message

implicit none

    character (len = 255) :: my_message

    write (*,'("Enter some text:")')
    read (*,'(A)') my_message
    
    write (*,'("Here is your text back:")')
    write (*,'(A)') my_message

end program Message
